src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java

```diff
@@ -46,6 +46,7 @@ public ReminderCommand parse(String args) throws ParseException {
             String[] stringArgsArr = trimmedArgs.replaceAll("\\s{2,}", " ").split(" ");
             long parsedNumber = Long.parseLong(stringArgsArr[0]);
             String timeUnit = stringArgsArr[1];
+            assert parsedNumber > 0 : "Input number must be a positive number";
             long numOfDaysFromToday = timeConversion(parsedNumber, timeUnit);
 
             if (trimmedArgs.isEmpty() || numOfDaysFromToday <= 0) {
```

Comment:
u did the assertion then here no need the if alr right :)

Reply:
I made changes to the assertion. It's no longer in this class hence the if  clause is still valid.